packages/geoprocessing/src/toolbox/geoblaze/rasterStats.ts

```diff
@@ -9,8 +9,20 @@ import {
   MetricDimension,
 } from "../../types/index.js";
 import geoblaze, { Georaster } from "geoblaze";
-import { StatsObject, CalcStatsOptions } from "../../types/geoblaze.js";
-import { toRasterProjection, defaultStatValues } from "./geoblaze.js";
+import {
+  StatsObject,
+  CalcStatsOptions,
+  Histogram,
+} from "../../types/geoblaze.js";
+import { toRasterProjection, geoblazeDefaultStatValues } from "./geoblaze.js";
+import cloneDeep from "lodash/cloneDeep.js";
+
+// default values for stats
+export const defaultStatValues = {
+  ...cloneDeep(geoblazeDefaultStatValues),
+  area: 0,
+  histogram: {},
+};
```

Comment:
I started with adding histogram, as well as area, to the list of stats that have default values initialized.